src/main/java/sirius/biz/importer/ImportContext.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.importer;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import sirius.db.jdbc.SQLEntity;
+import sirius.db.jdbc.batch.BatchContext;
+import sirius.db.jdbc.batch.FindQuery;
+import sirius.db.mixing.BaseEntity;
+import sirius.kernel.di.std.Parts;
+import sirius.kernel.health.Exceptions;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ImportContext {
+
+    private Importer importer;
+    private BatchContext batchContext;
+
+    @Parts(ImportHandler.class)
```

Comment:
@Parts kommt nicht mit listen klar da knallt es und des property ist null -> Collection oder PartCollection
(siehe PartsAnnotationProcessor)
Ändern oder erweitern. Des mit @Parts und List haben wir im übrigen auch in DistributedTasks einmal drin.